webserver/__init__.py

```diff
@@ -131,6 +134,17 @@ def before_request_gdpr_check():
         elif current_user.is_authenticated and current_user.gdpr_agreed is None:
             return redirect(url_for('index.gdpr_notice', next=request.full_path))
 
+    @app.before_request
+    def prod_https_login_redirect():
+        """ Redirect to HTTPS on the production login page
+        """
+        split_url = urlparse.urlsplit(request.url)
```

Comment:
just checking - what will this be when going through the MeB load balancers? I’ve not done it with flask before, but for example when I use loadbalancer https -> webserver http -> uwsgi -> django, the scheme is always http. @zas can you comment on what you think this might be for AB? Perhaps we can deploy a test app to check it.

How about doing this only when config.DEBUG is false?

Reply:
Internal network queries (between gateways & backend servers) are always unencrypted. Gateways are there to handle the SSL mess.
Else we need to manage certs to access to backends...